src/decouplededitorui.js

```diff
@@ -37,18 +37,21 @@ export default class DecoupledEditorUI extends EditorUI {
 	 */
 	init() {
 		const editor = this.editor;
-		const view = this.view;
+		const mainView = this.view;
```

Comment:
This is the view for the whole editor, not only for the main root, so `mainView` is missleading. Just `view` is better.